fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -16,6 +16,19 @@
 logger = logging.getLogger(__name__)
 
 
+class _h:
+    def __init__(self, classifier):
+        self._classifier = classifier
+
+    def __call__(self, X):
```

Comment:
It's always been sort of annoying that we don't call predict, but rather have a callable since it's inconsistent with scikit-learn. Maybe this is a good time to change that?

Reply:
We're already storing the classifier so perhaps we don't need more?

Reply:
That's a sentiment I have shared. This is a minimal change with the existing pattern, but.... I don't recall if there was a particular reason we kept to this when making all the other changes to ExpGrad.

Reply:
For me, the issue was that I didn't quite have enough time to sort out everything I didn't like :-) This is definitely one of those. @MiroDudik may have thoughts.

Reply:
yeah by looking at the code, it doesn't seem like something we need.

Reply:
If the reasons for needing are only historical, I'll happily change it to passing an estimator.... @MiroDudik ?